puma/yuma/__init__.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
```

Comment:
I would add the files in this dir to `hlplots` or alternatively move everything here to the `yuma` dir